Vision/Vision.bs

```diff
@@ -11,10 +11,10 @@ Previous Version: https://www.w3.org/TR/2024/NOTE-w3c-vision-20241018/
 Editor: Chris Wilson, Google, w3cid 3742
 Editor: Tantek Çelik, Mozilla, w3cid 1464
 Abstract:
-	The W3C Vision articulates W3C’s mission, 
+	The W3C Vision articulates the values that underpin W3C’s mission, 
 	what W3C is, what it does and why that matters,
-	and the values and principles by which it operates
-	and makes decisions.
+	and the principles by which
```

Comment:
I understand not repeating "values", but I think the "by which it operates" part is important.
 
```suggestion
	and the principles by which it operates
```